v3.1/custom-rules/before-crs.dist/baseconfig.conf

```diff
@@ -7,8 +7,8 @@ SecAction "id:900110,phase:1,pass,nolog,\
 SecAction "id:900120,phase:1,pass,nolog,\
   setvar:tx.paranoia_level=${PARANOIA}"
 
-SecAction "id:900200,phase:1,pass,nolog,setvar:tx.allowed_methods=${MODSEC_ALLOWED_METHODS}"
-SecAction "id:900220,phase:1,pass,nolog,setvar:tx.allowed_request_content_type=${MODSEC_ALLOWED_CONTENT}"
+SecAction "id:900200,phase:1,pass,nolog,setvar:'tx.allowed_methods=${MODSEC_ALLOWED_METHODS}'"
+SecAction "id:900220,phase:1,pass,nolog,setvar:'tx.allowed_request_content_type=${MODSEC_ALLOWED_CONTENT}'"
```

Comment:
That's interesting! Why is this quoting necessary? I did remove that, my bad!

Reply:
> Because the variables can contain white spaces.